cli/src/touch-dev-vars.ts

```diff
@@ -0,0 +1,32 @@
+import { existsSync, readFileSync, writeFileSync } from "node:fs";
+import { join } from "node:path";
+import type { Context } from "./context";
+
+/**
+ * Touch a .dev.vars file and add the default FPX_ENDPOINT environment variable if not present.
+ * @param context - The context object containing the project path.
+ */
+export function touchDevVars(context: Context): void {
+  if (!context.path) {
+    return;
+  }
+
+  try {
+    const projectDir = join(context.cwd, context.path);
+
+    const devVarsPath = join(projectDir, ".dev.vars");
+
+    let content = "";
+    if (existsSync(devVarsPath)) {
+      content = readFileSync(devVarsPath, "utf-8");
+    }
+
+    if (!content.includes("FPX_ENDPOINT=")) {
```

Comment:
okay possible issue with adding to dev.vars is that if they do not run studio, they'll get our really-hard-to-prevent exception logs that bubble up from the otel client library 

this could be bad dx, but! the logic of putting our url in dev vars immediately is that it reduces the chance of "oh you need to restart your app bc .dev.vars was updated" once fiberplane studio is conifgured.